tools/cert_create/Makefile

```diff
@@ -10,6 +10,7 @@ V		:= 0
 DEBUG		:= 0
 BINARY		:= ${PROJECT}${BIN_EXT}
 OPENSSL_DIR	:= /usr
+USE_TBBR_DEFS	?= 1
```

Comment:
I thought about it a little more.   Currently, the tool can be built directly in tools/cert_create if PLAT is passed from the command line.  As far as I understood, tools/cert_create/Makefile was implemented in that way.

After all I removed `USE_TBBR_DEFS ?= 1` entirely.   The direct build will be possible by passing correct combination PLAT and USE_TBBR_DEFS from the command line.  This does not look very nice, but we can excuse here.

I updated the user guide.